### DISCUSSION TITLE
"silkTouchNoContainerDropping"

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#74] [Ideas]
Allows to break containers like Chests, Hoppers, Droppers, etc and save their inventory data as NBT.

Great way to move Chests and containers in general.
Maybe expandable which Container blocks allowed and which not.

**Comment by ghost** [comment]
Maybe you can adapt code from Ivan Carpet Addition for it: 
https://github.com/Ivan-1F/Ivan-Carpet-Addition/blob/1.15.2/docs/rules.md#containerdropinventorydisabled

**Comment by wendavid552** [comment]
It will cause a lot of problems, such as item duping, and item sorter would be broken. Anyway, I agree that it could be an interesting feature, like what Mojang does in one block at a time, but it could be easily abused in survival.

**Comment by ghost** [comment]
Those items having special NBTs, mean they can't stacked with others.

Could you help me please with examples what could really go wrong?

In the end, I'm always on the site to let this behaviour a decision of the players. But maybe with a special warning message of possible known issues. If someone tries to activate the rule.

Ivan made this rule already - not sure how his experience with possible bugs was 🤔

**Reply by wendavid552** [reply]
For an example, just like stacking shulker boxes in Carpet-extra, a newly made hopper with no NBTs about its containing is unable to stack with another hopper that just cleaned up.

The reason why I do not like it is that it leaves illegal items in the world, which is not possible to achieve in any vanilla survival world.
